tests/components/airzone/test_config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Define tests for the Airzone config flow."""
+
+from unittest.mock import MagicMock, patch
+
+from aiohttp.client_exceptions import ClientConnectorError
+
+from homeassistant import data_entry_flow
+from homeassistant.components.airzone.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER, ConfigEntryState
+from homeassistant.const import CONF_HOST, CONF_PORT
+
+from .util import CONFIG, airzone_requests_mock
+
+from tests.common import MockConfigEntry
+from tests.test_util.aiohttp import mock_aiohttp_client
+
+
+async def test_form(hass):
+    """Test that the form is served with valid input."""
+
+    with patch(
+        "homeassistant.components.airzone.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry, mock_aiohttp_client() as _m:
```

Comment:
Its preferable to mock the pypi library instead of the http requests its making since the we want to verify the integrations interactions with the library.

The library should be testing the http requests

Reply:
Changed in 9e655b2693ded13aa0e4fa3de387da24e2c96cac